### PR TITLE
Add orderly blog post

### PR DESCRIPTION
**Pull request by richfitz**

### CODE REVIEW

**Review by r-ash — APPROVED** [review]
Looks great, that is a lot of changes!

File: content/blog/orderly-1.3.0.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+tags:
+ - orderly
+ - reproducible research
+---
+
+After 18 months of heavy use within [VIMC](https://www.vaccineimpact.org/) and in the [COVID-19 modelling response](https://www.imperial.ac.uk/mrc-global-infectious-disease-analysis/covid-19/covid-19-response-team-2020-2021-report/), we have released orderly 1.3.0 to CRAN. This is a major update with many new features (more than 100 PRs since version 1.0.4 was published in early 2020).
+
+Orderly is our [reproducible research framework](https://www.vaccineimpact.org/orderly/articles/orderly.html), designed to keep track of multiple versions of reports and allow researchers to automatically audit and roll back their data, or construct workflows around long running tasks.
+
+Since 2020 we've added:
```
Comment (weshinsley) [inline content/blog/orderly-1.3.0.md]: Consider chanhging "we've added" to "we have:" - and then the fixes below will make each sentence make sense if prefixed with that...

File: content/blog/orderly-1.3.0.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+* Expanded documentation, including [a discussion of different workflow patterns orderly enables](https://www.vaccineimpact.org/orderly/articles/patterns.html)
+* Support for exporting work to be rerun elsewhere (e.g., a HPC) and then sent back to a central server via [bundles](https://www.vaccineimpact.org/orderly/articles/bundles.html)
```
Comment (weshinsley) [inline content/blog/orderly-1.3.0.md]: Expanded *the* documentation

File: content/blog/orderly-1.3.0.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+* Massively improved the power of [OrderlyWeb](https://github.com/vimc/orderly-web/), which can be used as a central server to share artefacts among a team [see the remotes vignette](https://www.vaccineimpact.org/orderly/articles/remote.html).
+* Expanded how dependencies work, to allow depending on the latest version of a particular parameter (e.g., `latest(parameter:n_samples >= 1000)`) rather than just the latest version
+* Add new report development mode [`orderly_develop_start`, `orderly_develop_status` and `orderly_develop_clean`](https://www.vaccineimpact.org/orderly/reference/orderly_develop_start.html) which will setup environment so reports can be developed in the same was as one might write code outside of orderly. It copies required files, source code files, load declared packages. This supersedes `orderly_test_start` but function left in for now.
```
Comment (weshinsley) [inline content/blog/orderly-1.3.0.md]: *Added* new report...
Perhaps setup *an* environment
load -> *loads* declared packages
perhaps [the] function [is] left in for now

File: content/blog/orderly-1.3.0.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+* More easily depend on a mixture of draft and archive reports in development, via the `use_draft` argument to `orderly_run`
```
Comment (weshinsley) [inline content/blog/orderly-1.3.0.md]: (we have...)
Allowed easier dependency on a mixture of... ?

File: content/blog/orderly-1.3.0.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+* Visualise the graph structure of source or archive reports with [`orderly_graph`](https://www.vaccineimpact.org/orderly/reference/orderly_graph.html)
```
Comment (weshinsley) [inline content/blog/orderly-1.3.0.md]: Added tools to visualise

File: content/blog/orderly-1.3.0.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+* Helper functions, inspired by [`usethis`](https://usethis.r-lib.org/): [`orderly_use_resource` `orderly_use_source` and `orderly_use_package`](https://www.vaccineimpact.org/orderly/reference/orderly_use.html) which can add a resource, source, or package into the `orderly.yml`
```
Comment (weshinsley) [inline content/blog/orderly-1.3.0.md]: Added helper functions

File: content/blog/orderly-1.3.0.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+* Environment variables and secrets can be used in reports
```
Comment (weshinsley) [inline content/blog/orderly-1.3.0.md]: Allowed ... to be used

File: content/blog/orderly-1.3.0.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+
+In addition, there are lots of bug fixes and little features we've added as we need it, such as better handling of metadata for failed reports, more flexible querying of reports, integration with Microsoft Teams, more flexibility with complex SQL database configurations, and better error messages.
```
Comment (weshinsley) [inline content/blog/orderly-1.3.0.md]: perhaps "we've added as we need it" -> "as we needed them" or just "as needed"

File: content/blog/orderly-1.3.0.md [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+
+Over the course of the pandemic we have used orderly to collect together more than a terabyte of research outputs (mostly simulation data) among a distributed team. It has been key to many of our workflows over the last 18 months, and we hope that other groups can leverage this work.
```
Comment (weshinsley) [inline content/blog/orderly-1.3.0.md]: I _think_ Terabyte has a capital T?

**Review by weshinsley — COMMENTED** [review]
This looks great, and I think captures well how crucial orderly has been these months...

I have made some fiddly take/leave sort of grammary points, mostly to make the bullet points flow with the "We have:" in full sentences.

File: content/blog/orderly-1.3.0.md (hunk 2 above)
Comment (weshinsley) [reply content/blog/orderly-1.3.0.md]: _Added_ support for...

Also, possibly HPC you could call "compute cluster"

**Pull request merged**